amundsen_application/static/js/config/tests/index.spec.ts

```diff
@@ -27,6 +28,33 @@ describe('getDatabaseIconClass', () => {
   })
 });
 
+describe('getBadgeConfig', () => {
+  AppConfig.badges = {
```

Comment:
nit: Moving forward we should create fixtures and export/import this objects for tests as opposed to modifying out actual `AppConfig` object.